nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -410,6 +410,27 @@ in
           '';
       };
 
+    # FIXME: should move this to top-level.nix.
+    system.forbiddenDependencies = mkOption {
+      default = null;
+      example = "-dev$";
+      type = types.nullOr types.str;
+      description = ''
+        An extended regular expression that matches store paths that
+        should not appear in the system closure.
+      '';
+    };
+
+    system.maxClosureSize = mkOption {
+      default = null;
+      example = "-dev$";
```

Comment:
Shouldn't this give an example for the closure size?

Reply:
Indeed, thanks!